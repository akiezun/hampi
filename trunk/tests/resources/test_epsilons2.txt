S = A B "c" ;
A = | "a" ;
B = C | ;
C = ;