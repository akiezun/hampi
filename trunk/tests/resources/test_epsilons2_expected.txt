A = "a"  ;
S = "c"  | A "c"  ;